src/lib/components/workers-table.svelte

```diff
@@ -0,0 +1,17 @@
+<section
+  class="flex flex-col border-2 border-gray-300 w-full rounded-lg workers-table"
+>
+  <div class="bg-gray-200 flex flex-row p-2">
+    <div class="w-3/12 text-left">ID</div>
+    <div class="w-3/12 text-left">Last Accessed</div>
+    <div class="w-3/12 text-left">Workflow Table Handler</div>
+    <div class="w-2/12 text-left">Activity Handler</div>
+  </div>
+  <slot />
+</section>
+
+<style lang="postcss">
+  .workers-table {
+    height: calc(100vh - 280px);
```

Comment:
Are we using a Virtual list here? If not, I'm not sure we need this. Is the 280px coming from anywhere in particular?